mix.exs

```diff
@@ -33,7 +33,8 @@ defmodule Ueberauth.Okta.Mixfile do
       {:oauth2, "~> 2.0"},
       {:ueberauth, "~> 0.7"},
       {:credo, "~> 1.5", only: [:dev, :test]},
-      {:ex_doc, ">= 0.0.0", only: :docs, runtime: false}
+      {:ex_doc, ">= 0.0.0", only: :docs, runtime: false},
```

Comment:
I think this should be `only: :dev` 🤔 . Or maybe use the latest suggestion from their readme:
```elixir
{:ex_doc, "~> 0.27", only: :dev, runtime: false}
```
How that sounds?

Reply:
oh sure yeah